### PR TITLE
feat: redux tests pt 1

### PR DESCRIPTION
**Pull request by muselesscreator**
Adds tests for top-level redux, app reducer and selectors, requests selectors, and app thunkActions.

### CODE REVIEW

File: src/editors/data/redux/app/selectors.test.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  });
+  describe('returnUrl', () => {
+    it('is memoized based on unitUrl and studioEndpointUrl', () => {
+      expect(selectors.returnUrl.preSelectors).toEqual([
+        simpleSelectors.unitUrl,
+        simpleSelectors.studioEndpointUrl,
+      ]);
+    });
+    it('returns urls.unit with the unitUrl if loaded, else an empty string', () => {
+      const { cb } = selectors.returnUrl;
```
Comment (connorhaugh) [inline src/editors/data/redux/app/selectors.test.js:57]: Just curious, what does "cb" refer to as an acronym

**Review by connorhaugh — CHANGES_REQUESTED** [review]
Looks good, just one question on nomenclature. Otherwise, if we're going to trim down lint errors: These two are from files this PR modifies: 
`/home/runner/work/frontend-lib-content-components/frontend-lib-content-components/src/editors/data/redux/app/selectors.test.js
   2:10  error  'blockTypes' is defined but never used        no-unused-vars
  15:7   error  'testKey' is assigned a value but never used  no-unused-vars`

and 

'/home/runner/work/frontend-lib-content-components/frontend-lib-content-components/src/editors/data/redux/thunkActions/app.test.js
  1:10  error  'locationId' is defined but never used  no-unused-vars'

File: src/editors/data/redux/app/selectors.test.js (hunk 1 above)
Comment (muselesscreator) [reply src/editors/data/redux/app/selectors.test.js:57]: callback

**Review by connorhaugh — APPROVED** [review]
LTGM tysm for the lint fixes

File: src/editors/data/redux/app/reducer.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { initialState, actions, reducer } from './reducer';
+
+const testingState = {
+  ...initialState,
+  arbitraryField: 'arbitrary',
+};
+
+describe('requests reducer', () => {
```
Comment (connorhaugh) [inline src/editors/data/redux/app/reducer.test.js:8]: this is app reducer, not request reducer-- just commenting because I saw this, it is just a nit!

Comment (muselesscreator) [reply src/editors/data/redux/app/reducer.test.js:8]:
```suggestion
describe('app reducer', () => {
```

**Pull request merged**